microsoft/electionguard-python
# #754 Multiple Uploads

### Issue

Fixes #754 

### Description
This PR attempts several UX enhancements in order to clarify how multiple ballots works.  In particular:

* Renames "Decryption" to "Tally" in the UI.  In other words, to decrypt votes and view a tally and spoiled ballots you now create a new Tally rather than creating a Decryption.
* Excludes duplicate ballots.  If you upload the same ballots multiple times the UI will warn you.
* Displays a sum of ballots uploaded on the election page
* Shows the number of ballots uploads and the sum of ballots for each tally
* Replaces the individual buttons with an action button and dropdown menu to show descriptions next to each action

### Testing
Go to the election page, it should now look like this:

![image](https://user-images.githubusercontent.com/1769905/186018652-fa172e62-a0e6-4673-bde2-7fb3414dc75c.png)

* Notice "Decryptions" in the bottom left is now called "Tallies"
* Notice the sum of all successfully uploaded ballots (81) in the bottom left
* Notice the action button in the top right.  If you click it you get this:

![image](https://user-images.githubusercontent.com/1769905/186018762-404e9ca5-f9e5-4995-b6c4-13e285745979.png)

Notice "Add Tally" not "Add Decryption"

If you try to add ballots multiple times you get this while uploading:

![image](https://user-images.githubusercontent.com/1769905/186019009-56aee653-0767-401f-a10d-f67d158faa41.png)

And if there were no new ballots you'll get this:

![image](https://user-images.githubusercontent.com/1769905/186019068-a787ae39-d8b3-4b38-b5ed-77127a49d9b2.png)

Finally, when you create a Tally, it now shows the number of uploads and the number of ballots it contains:

![image](https://user-images.githubusercontent.com/1769905/186019791-84e566c1-1759-4e0f-8c16-e422ebed1a6f.png)
